homeassistant/components/squeezebox/button.py

```diff
@@ -153,6 +153,11 @@ def __init__(
             f"{format_mac(self._player.player_id)}_{entity_description.key}"
         )
 
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        return self.coordinator.available and super().available
```

Comment:
I'm not fully following this one:

![CleanShot 2025-05-11 at 20 55 41@2x](https://github.com/user-attachments/assets/41f1318f-e3a7-40a2-a959-44b0d2e4f5bf)

The parent entity of this entity already is a CoordinatorEntity, so it already takes the coordinator into account?

../Frenck